packages/telemetry/src/slog-to-otel.js

```diff
@@ -106,6 +110,57 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
   const dbTransactionManager = makeKVDatabaseTransactionManager(db);
   const kernelPromiseToSendingCause = makeKVStringStore('kernelPromise', db);
 
+  /** @type {WeakMap<Promise | object, {kref: string, iface: string | undefined}>} */
+  const presenceMap = new WeakMap();
+  /** @type {Map<string, Promise | object>} */
+  const valCache = new Map();
+
+  const slotToVal = (kref, iface) => {
```

Comment:
why does handling smallcaps need the whole `slotToVal` machinery? How did we get by without it before?

Reply:
is this just handling smallcaps in the way we did before? Or is it parsing the whole message in a more fine-grain way or something?

Reply:
It was not handling smallcaps at all before, just throwing because `body` started with a `#` which is invalid JSON. 

Before we were manually parsing `body` assuming it was a JSON serialized string, more precisely a qclass based caps encoding.

This is the same kind of change that the kernel went through in https://github.com/Agoric/agoric-sdk/pull/6450, and uses a dummy `slotToVal` converter.